### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by KazuCocoa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+**Please consider moving to [appium-mac2-driver](https://github.com/appium/appium-mac2-driver) for newer macOS versions**
```
Comment (mykola-mokhnach) [inline README.md:1]: I would rather explicitly mention which versions (e.g. macOS 10.15 Catalina or newer)

Comment (mykola-mokhnach) [reply README.md:1]: Also, it makes sense to mention that the Appium team is not going to continue the further development of this driver

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+**Please consider moving to [appium-mac2-driver](https://github.com/appium/appium-mac2-driver) for over macOS 10.15)**
+
+**Appium team will not contribute this driver further**
```
Comment (mykola-mokhnach) [inline README.md:3]: The Appium team ... contribute to ...

File: README.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+**Please consider moving to [appium-mac2-driver](https://github.com/appium/appium-mac2-driver) for over macOS 10.15)**
```
Comment (mykola-mokhnach) [inline README.md:1]: consider switching to

**Review by mykola-mokhnach — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+**Please consider switching to [appium-mac2-driver](https://github.com/appium/appium-mac2-driver) for over macOS 10.15)**
```
Comment (mykola-mokhnach) [inline README.md:1]: Extra parenthesis at the end?

Comment (KazuCocoa) [reply README.md:1]: oh..

**Pull request merged**